virttest/libvirt_xml/vm_xml.py

```diff
@@ -557,18 +557,21 @@ def __init__(self, hypervisor_type='kvm', virsh_instance=base.virsh):
         self.xml = u"<domain type='%s'></domain>" % hypervisor_type
 
     @staticmethod  # static method (no self) needed b/c calls VMXML.__new__
-    def new_from_dumpxml(vm_name, options="", virsh_instance=base.virsh):
+    def new_from_dumpxml(vm_name, options="", virsh_instance=base.virsh,
```

Comment:
@balamuruhans 
Sorry, I'm not that clear about your meaning. If you want to operate xml object of remote machine, just use create a remote virsh instance and pass it to this method is not enough?